modular/gater/auth_handler.go

```diff
@@ -5,10 +5,11 @@ import (
 	"encoding/xml"
 	"net/http"
 	"regexp"
-	"strconv"
 	"strings"
 	"time"
 
+	"github.com/bnb-chain/greenfield-storage-provider/util"
```

Comment:
the imports should be formatted by stdlib/third party/current_project

Reply:
Done